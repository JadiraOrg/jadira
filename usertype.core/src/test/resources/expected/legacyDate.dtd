<!ELEMENT dataset (
    LEGACYDATE*)>

<!ELEMENT LEGACYDATE EMPTY>
<!ATTLIST LEGACYDATE
    ID CDATA #REQUIRED
    LEGACYDATE CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

