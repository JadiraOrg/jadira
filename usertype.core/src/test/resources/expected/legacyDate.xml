<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "legacyDate.dtd">
<dataset>
  <LEGACYDATE ID="0" LEGACYDATE="2004-02-25 12:11:10.0" NAME="test_0"/>
  <LEGACYDATE ID="1" LEGACYDATE="1980-03-11 13:12:11.0" NAME="test_1"/>
</dataset>